src/ui/public/courier/data_source/_abstract.js

```diff
@@ -1,14 +1,17 @@
 import _ from 'lodash';
 import angular from 'angular';
-import CourierRequestQueueProvider from 'ui/courier/_request_queue';
-import CourierErrorHandlersProvider from 'ui/courier/_error_handlers';
-import CourierFetchFetchProvider from 'ui/courier/fetch/fetch';
-import CourierDataSourceDecorateQueryProvider from 'ui/courier/data_source/_decorate_query';
+
+import 'ui/promises';
```

Comment:
Hmm... how was this service working before then?  I'd err on the side of not including this unless we're certain it's necessary.